CHANGELOG.md

```diff
@@ -1,30 +1,20 @@
 # Changelog
 
-### v4.0.0-rc.0 (2024-05-03)
+See the [upgrading to v4](/docs/upgrading-to-v4.md) documentation for a complete list of all API changes in version 4.
 
-- **[BREAKING]** Update types to support more generic usage
-  ([#471](https://github.com/GoogleChrome/web-vitals/pull/471))
-- Fix `reportAllChanges` behavior for LCP when library is loaded late ([#468](https://github.com/GoogleChrome/web-vitals/pull/468))
-
-### v4.0.0-beta.3 (2024-04-22)
+### v4.0.0 (2024-05-13)
```

Comment:
FYI, I put `2024-05-13` assuming that's when we'll release, but we can change it if needed.